sktime/transformations/compose.py

```diff
@@ -62,39 +62,42 @@ class TransformerPipeline(BaseTransformer, _HeterogenousMetaEstimator):
     """Pipeline of transformers compositor.
 
     The `TransformerPipeline` compositor allows to chain transformers.
-    The pipeline is constructed with a list of sktime transformers,
-        i.e., estimators following the BaseTransformer interface.
-    The list can be unnamed - a simple list of transformers -
-        or string named - a list of pairs of string, estimator.
+    The pipeline is constructed with a list of sktime transformers, i.e.
+    estimators following the BaseTransformer interface. The list can be
+    unnamed (a simple list of transformers) or string named (a list of
+    pairs of string, estimator.
 
     For a list of transformers `trafo1`, `trafo2`, ..., `trafoN`,
-        the pipeline behaves as follows:
-    `fit` - changes state by running `trafo1.fit_transform`, `trafo2.fit_transform`, etc
-        sequentially, with `trafo[i]` receiving the output of `trafo[i-1]`
-    `transform` - result is of executing `trafo1.transform`, `trafo2.transform`, etc
-        with `trafo[i].transform` input = output of `trafo[i-1].transform`,
-        and returning the output of `trafoN.transform`
-    `inverse_transform` - result is of executing `trafo[i].inverse_transform`,
-        with `trafo[i].inverse_transform` input = output `trafo[i-1].inverse_transform`,
-        and returning the output of `trafoN.inverse_transform`
-    `update` - changes state by chaining `trafo1.update`, `trafo1.transform`,
-        `trafo2.update`, `trafo2.transform`, ..., `trafoN.update`,
-        where `trafo[i].update` and `trafo[i].transform` receive as input
+    the pipeline behaves as follows:
+
+        * `fit` - changes state by running `trafo1.fit_transform`,
```

Comment:
this renders strangly in readthedocs. Is the current rendering not ok?

Reply:
ther current rendering is weird bcz its showing your list on same level as the ``Parameters` and `Attributes` section:
![image](https://user-images.githubusercontent.com/29627036/189611354-03cd8a42-20c3-4af8-9e78-ec90870882b8.png)